participant/app/routes/$localAgency/recertify/index.tsx

```diff
@@ -27,6 +28,7 @@ export const loader: LoaderFunction = async ({
     { headers: headers }
   );
 };
+const feedbackAlertHeading = <Trans i18nKey="Index.feedback.heading" />;
```

Comment:
Could just set this in the prop? 🤷🏻‍♀️

Reply:
Hmmmm I tried this yesterday and it wasn't working, but now it is, so I've incorporated it 🤷 . I must have done something wrong yesterday.